1_crosswalk_fetch.yml

```diff
@@ -43,3 +44,10 @@ targets:
       '1_crosswalk_fetch/out/lakes_shapefile.dbf.ind',
       '1_crosswalk_fetch/out/lakes_shapefile.prj.ind',
       '1_crosswalk_fetch/out/lakes_shapefile.shx.ind')
+  1_crosswalk_fetch/out/lakes_sf.rds:
```

Comment:
How does this file get created?

Reply:
i think the lines just got cut off. this gets created by the recipe just above this one, https://github.com/USGS-R/lake-temperature-model-prep/pull/12/files/bef264855be80707b3915cda60aa4d8f5e0efe8b#diff-99ca52bbbb504932bc99ac5eceed3bd2R40
```
   1_crosswalk_fetch/out/lakes_sf.rds.ind:
     command: munge_crosswalk(
       out_ind=target_name,
       shp_ind='1_crosswalk_fetch/out/lakes_shapefile.shp.ind',
       '1_crosswalk_fetch/out/lakes_shapefile.dbf.ind',
       '1_crosswalk_fetch/out/lakes_shapefile.prj.ind',
       '1_crosswalk_fetch/out/lakes_shapefile.shx.ind')
```